base/exports.jl

```diff
@@ -1138,9 +1139,7 @@ export
     listenany,
     ltoh,
     mark,
-    mmap_array,
-    mmap_bitarray,
-    msync,
+    mmap,
     nb_available,
```

Comment:
Ah, that makes sense. I think I just needed to export `msync` and just `mmap_array` on windows.